src/index.ts

```diff
@@ -342,18 +352,32 @@ export default class EditorjsList {
        * For each counter type in OlCounterType create toolbox item
        */
       OlCounterTypesMap.forEach((_, counterType: string) => {
+        const counterTypeValue = OlCounterTypesMap.get(counterType)! as OlCounterType;
+
+        if (!this.defaultCounterTypes.includes(counterTypeValue)) {
+          return;
+        }
+
         orderedListCountersTunes.children.items!.push({
           title: this.api.i18n.t(counterType),
-          icon: OlCounterIconsMap.get(OlCounterTypesMap.get(counterType)!),
+          icon: OlCounterIconsMap.get(counterTypeValue),
           isActive: (this.data.meta as OrderedListItemMeta).counterType === OlCounterTypesMap.get(counterType),
           closeOnActivate: true,
           onActivate: () => {
             this.changeCounters(OlCounterTypesMap.get(counterType) as OlCounterType);
           },
         });
       });
+
+      /**
+       * Dont show Counter type tune if there is no valid counter types
+       */
+      if (orderedListCountersTunes.children.items!.length > 0) {
```

Comment:
if we will have 1 counter type, than it would be information plate, not a tune, better to add tunes, when user could change smth
```suggestion
      if (orderedListCountersTunes.children.items!.length > 1) {
```

Reply:
yeah it make sense. kindly review the PR again thanks @e11sy